### ISSUE TITLE
720p for switch

### ISSUE THREAD

**Issue by ghost** [GhostlyDark/OoT-Reloaded-SoH#17]
Is there anyway a 720p version could be made

I play ship of harkinian on switch and the 720p resolution of the switch makes the 1080p textures unnecessary to use compared to a 720p version. also it might help with the switches low memory.

**Comment by GhostlyDark** [comment]
Is performance an issue on Switch with the downscaled release? Currently, it uses a cap of 8x the original texture resolution, which could be lowered to 4x without much effort. 

The only reason I don't upload the files is simply because this adds to the burden of having to compile and upload yet another set of files for each and every release.

This is something that is being requested frequently though, so I am considering it for the next release.

**Comment by Dr01000111** [comment]
Yes. it adds significant stuttering anytime a action is done like hitting a enemy or climbing a wall. it also is maxing out the memory and is only somewhat decent fps when setting the ram clock speeds past safe values using a modified version of sysclk.

**Comment by beyondmeat** [comment]
There is also a filesize limit issue since most people use fat32.

**Comment by Xaero252** [comment]
I can also confirm that the current lowest resolution release is a bit of a performance impact on switch. In some areas there's no real impact but in others the game has significant performance problems, audio degradation due to the framerate dropping etc.

The Switch obviously isn't the target hardware for this project - and both the IOPS and total throughput of an SD card, in combination with the limited total memory mean that textures targeted at 720p are probably the best the hardware can handle.
